src/common/components/form/validationUtils.tsx

```diff
@@ -0,0 +1,57 @@
+import { formatMessage } from '../../translation/utils';
+import { newMoment } from '../../time/utils';
+import {
+  SUPPORTED_START_BIRTHDAY,
+  DEFAULT_DATE_FORMAT,
+} from '../../time/constants';
+
+/** validateBirthDay()
+ * Validate user input child birthday.
+ * This app only target recently born child, so we support child which born after 2019 only.
+ * Input time which come from future is not accepted
+ * @param value Input value.
+ */
+const validateBirthDay = (value: string | number | undefined) => {
+  if (!value) {
```

Comment:
How can we test this service before the year 2020?

Reply:
i thought we only support kids born from beginning of 2019 -> now, not in future or 2018?